Assets/HoloToolkit/Utilities/Scripts/Solvers/Solver.cs

```diff
@@ -99,41 +107,63 @@ protected virtual void Start()
 
         protected IEnumerator CoStart()
         {
-                switch (solverHandler.TrackedObjectToReference)
-                {
-                    case SolverHandler.TrackedObjectToReferenceEnum.Head:
-                        while (CameraCache.Main == null || CameraCache.Main.transform == null)
-                        {
-                            yield return null;
-                        }
-                        //Base transform target to camera transform
+            // TODO: This really belongs in SolverHandler, everything in here is just passing SolverHandler member variables to SolverHandler functions. 
```

Comment:
should we go ahead and move it?

Reply:
I have a refactor in progress that resolves some of this. I'll retarget it for this PR and work with @Ecnassianer to get it all in at once.